homeassistant/components/nut/strings.json

```
@@ -75,6 +75,10 @@
       "test_panel_start": "Start testing the UPS panel",
       "test_panel_stop": "Stop a UPS panel test",
       "test_system_start": "Start a system test"
```

Missing comma in json file:

```suggestion
      "test_system_start": "Start a system test",
```